src/blockchain/ada.ts

```diff
@@ -0,0 +1,58 @@
+import { get, post } from '../connector/tatum';
+import { CardanoBlockChainInfo, TransactionHash, CardanoAccount, CardanoBlock, CardanoTransaction, CardanoGenerateWallet } from '../model';
+
+/**
+ * For more details, see <a href="https://tatum.io/apidoc#operation/CardanoBroadcast" target="_blank">Tatum API documentation</a>
+ */
+export const cardanoBroadcast = async (txData: string): Promise<TransactionHash> =>
+  post(`/v3/cardano/broadcast`, { txData });
+
+/**
+ * For more details, see <a href="https://tatum.io/apidoc#operation/CardanoGetBlockChainInfo" target="_blank">Tatum API documentation</a>
+ */
+export const cardanoGetBlockChainInfo = async (): Promise<CardanoBlockChainInfo> => get(`/v3/cardano/info`);
+
+/**
+ * For more details, see <a href="https://tatum.io/apidoc#operation/CardanoGenerateWallet" target="_blank">Tatum API documentation</a>
+ */
+export const cardanoGenerateWallet = async (mnemonic?: string): Promise<CardanoGenerateWallet> =>
```

Comment:
this should not be here, because this method contains sensitive information - mnemonic or private key

Reply:
Done.